lib/galaxy/selenium/navigation.yml

```diff
@@ -70,6 +70,7 @@ masthead:
 
 preferences:
   selectors:
+    sign_out: "#edit-preferences-custom-builds"
```

Comment:
This ID looks wrong. custom-builds should not be the ID for sign-out.

Reply:
I will look for the proper way to change the id and fix it. I will also check the other ids on that page to see if its the only one.

Reply:
Id has been changed, and I will ask @OlegZharkov to double check that there is no side effect.